documentation/final-deliverables/pdf-report.md

```diff
@@ -105,16 +105,55 @@ FR23. To "unmatch" Mentees and Mentors, the Mentee can communicate with the Ment
+
+- The ‘Authentication’ tab of the Firebase console allows us to verify Mentor and Admin identities upon account creation as well as to validate credentials upon login. All users are required to verify their email address before being able to access Admin or Mentor privileges of the website.
+  - Our Firebase configuration uses the ‘Email/Password’ Sign-in method. This can be modified if needed in the future by going to the ‘Sign-in method’ tab under ‘Authentication’.
+  - Authentication also handles password reset changes.
+
+Disclaimer: Our application does not have a UI to remove users. If a user is to be removed, go to the ‘Authentication’ tab of Firebase, find the user’s email address, then click on the three vertical dots. Simply click ‘Delete Account’ to remove that user. To ensure that the database isn’t cluttered, it would also be a good idea to remove that user from the Realtime Database.
 
 <br>
 
 ## Limitations of MentorMatch (and future improvements)
 
-* The way a mentor's availability is obtained is through a Google Calendar link field on the mentor form. This field is currently limited to only accepted Google Calendar links.
+- The way that a mentor's availability is obtained is through a url input field on the Mentor form. This field is currently limited to only accept Google Calendar links.
 
-* Mentees and mentors communicate via email messages. Due to this, if MentorMatch was further developed and internal communication between mentor and mentee were created, mentees would also need a MentorMatch account.
+- Mentees and Mentors communicate via email messages externally from MentorMatch. Due to this, if MentorMatch was further developed to require internal communication between Mentors and Mentees, Mentees would also need a MentorMatch account.
 
-* Mentors will have to manually update their calendar to reflect booked timeslots with Google Calendars. *Note*: Google Appointment Slots helps to remediate this, but since we don't have access to an organization or school Google Calendar, it is not something that we are able to test. However, organizations will likely be able to use Appointment Slots.
+- Mentors will have to manually update their calendar to reflect booked timeslots with Google Calendars. _Note_: (Google Appointment Slots)[https://support.google.com/calendar/answer/190998?co=GENIE.Platform%3DDesktop&hl=en] would help to remediate this, but since we don't have access to an organization or school Google Calendar, it is not something that we are able to test. However, organizations will likely be able to use Appointment Slots.
 
-* In the case that MentorMatch grows and needs to send >500 emails per day, Nodemailer will became a limitation.
+- In the case that MentorMatch grows and needs to send >500 emails per day, Nodemailer will become a limitation.
```

Comment:
@jpabadir I realized after I made these changes that you'll be working on the '## Limitations of MentorMatch (and future improvements)' of the document. I hope that the changes that I've added don't over-step too much on your task!

Reply:
This is great, Katrina. Thanks for helping me get ahead :)